packages/web3-constants/evm/debank.json

```
@@ -17,6 +17,8 @@
         "Celo": "celo",
         "Fantom": "ftm",
         "Aurora": "aurora",
-        "Aurora_Testnet": ""
+        "Aurora_Testnet": "",
+        "Harmony": "",
```

```suggestion
        "Harmony": "hmy",
```

Go to https://debank.com/profile/0xd8da6bf26964af9d7eed9e03e53415d37aa96045, and checkout the Network panel of devtools.